incubator/couchdb/templates/configmap.yaml

```diff
@@ -8,6 +8,7 @@ metadata:
     heritage: {{ .Release.Service | quote }}
     release: {{ .Release.Name | quote }}
 data:
+  clusterSize: "{{ .Values.clusterSize }}"
```

Comment:
is this variable added to values.yaml and README?

Reply:
Yes.

Reply:
Sorry for not checking myself.

Reply:
No problem!